source/GameWindow.cpp

```diff
@@ -233,7 +235,7 @@ bool GameWindow::Init()
 	io.ConfigFlags |= ImGuiConfigFlags_NavEnableKeyboard;
 
 	ImGui_ImplSDL2_InitForOpenGL(mainWindow, context);
-	ImGui_ImplOpenGL3_Init(nullptr);
+	ImGui_ImplOpenGL3_Init("#version 100");
```

Comment:
I don't think this change is needed anymore.

Reply:
I found I still needed it to make the imgui gui show up, I stuck it in an `#ifdef __EMSCRIPTEN__` for now

Reply:
That is very surprising. I'll have debug this then (if it's even possible?). The functions uses `"#version 100"` if emscripten is used so yeah very surprising.

Reply:
Works for me using the ES 3.